packages/amazonq/.changes/next-release/Feature-7ffbdb5d-40c5-4c3e-bc20-63d731bc51d2.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Uses AB variation as the name for overridden customizations"
```

Comment:
Thanks for including a changelog. Please review the [changelog guidelines](https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog) and adjust this based on the advice there.

Reply:
Better with the latest update?

Reply:
probably use `fix(amazonq): Using AB variation value as customization name when overridden`

Reply:
ref: `type: must be a valid type (build, ci, config, deps, docs, feat, fix, perf, refactor, style, telemetry, test, types)`